README.rst

```diff
@@ -51,6 +51,14 @@ System requirements:
 - gobject-introspection type libraries for GTK+, Pango
 - GTK+ 3.10 or newer (3.14 or newer for best results)
```

Comment:
I suspect this is no longer true and current gtimelog needs something newer.

I'd have to check.